src/lib/numa.lua

```diff
@@ -28,12 +29,24 @@ function cpu_get_numa_node (cpu)
 end
 
 function has_numa ()
-   local node1 = S.open('/sys/devices/system/node/node1', 'rdonly, directory')
+   local node1 = S.open(node_path..tostring(1), 'rdonly, directory')
    if not node1 then return false end
    node1:close()
    return true
 end
 
+-- Return the first and last CPU ids (as numbers) on the given NUMA node.
+function get_cpus_per_numa_node (node)
```

Comment:
Yes, I thought it could be useful outside of tests. I'll document it gladly, where shall I?

Oh, if the list of CPUs is not contiguous, that's a problem indeed. But we're recommeding turning off hyperthreads anyway, aren't we? ;-)

I wonder if there are any docs for that part of the kernel's `/sys/` tree, or something from Intel about this...